### PR TITLE
Make primaryServerQuery public

### LINKED ISSUE #8923: Make primaryServerQueryTimeInterval public on the LdapRegistry
The fail back property `returnToPrimaryServer ` is public, but the companion property `primaryServerQueryTimeInterval ` is currently internal. We should also expose `primaryServerQueryTimeInterval ` and add some tests to confirm that both configuration settings are working.

### PR DESCRIPTION
**Pull request by agiridharan**
fixes #8923

### CODE REVIEW

**Comment by andrea-pichler** [comment]
No message review requested from L2

**Comment by agiridharan** [comment]
#build

**Comment by agiridharan** [comment]
#build

**Comment by agiridharan** [comment]
#build

**Comment by agiridharan** [comment]
#build

1.

**Comment by agiridharan** [comment]
#build

**Comment by agiridharan** [comment]
ID Team, please review the added name and description of primaryServerQueryTimeInterval to the NLS file `dev/com.ibm.ws.security.wim.adapter.ldap/resources/OSGI-INF/l10n/metatype.properties`

File: dev/com.ibm.ws.security.wim.adapter.ldap/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -411,6 +411,9 @@ referral.desc=Specify the behavior for LDAP referrals. The default behavior is t
 referral.ignore=Ignore LDAP referrals.
 referral.follow=Follow LDAP referrals.
 
+primaryServerQueryTimeInterval=Primary server query time interval
+primaryServerQueryTimeInterval.desc=Polling interval for testing the primary server availability(in minutes)
```
Comment (dmuelle) [inline dev/com.ibm.ws.security.wim.adapter.ldap/resources/OSGI-INF/l10n/metatype.properties]: Change
Polling interval for testing the primary server availability(in minutes)
To
The polling interval, in minutes, at which the primary server is tested for availability.

Is it possible to say what component or process is testing the server for availability? For example

The polling interval, in minutes, at which (X) tests the primary server for availability.

**Pull request merged**